pyproject.toml

```diff
@@ -26,6 +26,8 @@ full = [
     "mlflow>=3.0.0",
     "wandb",
     "tqdm>=4.66.1"
+    # Pull Opik Optimizer from the branch that includes the native adapter
+    "opik_optimizer @ git+https://github.com/comet-ml/opik.git@vk/gepa-native-adapter#subdirectory=sdks/opik_optimizer"
```

Comment:
Why is this required and is there any way to avoid adding new dependencies?

Reply:
@LakshyAAAgrawal i will remove this, i need this for testing (for now) hence why PR in draft. If you can sense check the rest of the adapter I will clean this all up and finalize the PR as i need to also do a PR on my side to tie everything togther.